Prolangle/Services/GuessGame.cs

```diff
@@ -1,24 +1,40 @@
 ﻿using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using Prolangle.Languages.Framework;
 
 namespace Prolangle.Services;
 
 public class GuessGame
 {
-	public GuessGame(LanguagesProvider provider, ILogger<GuessGame> logger, Func<int> seeder,
+	public Func<int> Seeder { get; }
+
+	public GuessGame(LanguagesProvider provider, LanguageSnippetProvider snippetProvider,
+		ILogger<GuessGame> logger, GameSeeder seedGenerator,
 		IWebAssemblyHostEnvironment environment)
 	{
-		var languages = provider.Languages;
+		Seeder = seedGenerator.Seeder;
+
+		MetadatumGameLanguage = _PickRandomLanguage(provider.Languages);
```

Comment:
If we make the snippet game provide its own supported languages, I think it would be sensible to also make the metadatum game provide its supported languages. That we we can add languages to each independently.

Reply:
We could, but the metadatum game doesn't really exist as its own service at this point.

Reply:
I see. We should do that in the near future, I think.